src/api/movie.ts

```diff
@@ -1,16 +1,29 @@
 import axios from 'axios'
 
-const url2 = 'https://tinfo.apiumando.info/3/movie/'
-const API_KEY = '49101d62654e71a2931722642ac07e5e'
-const yts = 'https://yts.am/api/v2/'
+const API_URL = 'https://api.themoviedb.org/3/'
+const API_KEY = '3c817edb2c2a10ea7095bc697864900d'
+const TORRENT_API_URL = 'https://yts.lt/api/v2/'
+
+function fetch(url: string, params?: any) {
+  return axios.get(`${API_URL}${url}`, {
+    params: {
+      api_key: API_KEY,
+      language: 'en-US',
+      ...params,
+    },
+  })
+}
 
 export default {
   movies(url: string) {
     return {
-      getByPage: (sort = 'seeds', index: number) =>
-        axios.get(
-          `${url}list?sort=${sort}&short=1&cb=&quality=720p,1080p,3d&page=${index}`,
-        ),
+      getByPage: (sort = 'seeds', index: number) => {
```

Comment:
It would be super helpful to type the return type of each of the `get*` functions that you have defined here.

Reply:
Will do it to another methods in the future when I use them.